src/NuGet.Clients/NuGet.CommandLine/CommandLineMachineWideSettings.cs

```diff
@@ -8,17 +8,23 @@
 namespace NuGet.CommandLine
 {
     [Export(typeof(Configuration.IMachineWideSettings))]
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
We've historically added our suppressions in a GlobalSuppressions file, since this makes the file really busy.

Reply:
It didn't give me the option to in the "Quick actions" even after creating a global suppression file. Lemme see just force the value in there and see if it works

Reply:
I 100% disagree with putting all the suppressions in a GlobalSuppressions file. It being busy in the code is a feature, because when I'm reviewing a future PR and I see they're modifying a line within a `#pragma disable` block, or a line right next to it, I can comment on the PR saying please remove this tech debt.  If the suppression is in the GlobalSuppresions file, it becomes much more difficult to know that there's an analyzer warning that's being suppressed and that improvements could be made.

If this is specifically about VS1591 or other XML comment warnings, then maybe we should disable this analyzer for proejcts that don't ship to nuget.org as packages.

I just don't see GlobalSuppressions.cs as a positive thing, with the possible exception of WPF code and the "keep the object heirarchy lower than X" warning.